docs/customizations.md

```diff
@@ -25,6 +25,21 @@ For instance the variable `proftpd_nat_masquerade` is set to `false` in `group_v
 apply the `[aws]` group variables which set `proftpd_nat_masquerade` to true, so that hosts in the aws group will have
 this aws-specific setting applied. Any combination of groups may be used.
 
+If you want to apply any of the changes you made to the variables you need to run the playbook again, making sure that
+the host you are targeting is in the right group. The simplest way to do so is to use an inventory file that only contains
+the group and the host you wish to target. If this is for example the group metavisitor, and you target the host localhost,
+your inventory file should look like this:
+
+```
+[metavisitor]
```

Comment:
Did you try in the vagrant example ?
It does not work for me
```
[artimed]
#localhost ansible_ssh_user="root" ansible_ssh_private_key_file="~/.ssh/id_rsa"
[metavisitor]
localhost
#ansible_ssh_user="root" ansible_ssh_private_key_file="~/.ssh/id_rsa"
[travis_bioblend]
localhost ansible_connection=local
[aws]
# Put you aws IP and key here to make FTP work in the default VPC.
# If you want further group-specific variables, put the host in these groups as well [e.g artimed].
```
and
chris$ ansible-playbook --inventory-file=hosts galaxy.yml
returns
```
[DEPRECATION WARNING]: Instead of sudo/sudo_user, use become/become_user and make sure become_method is 'sudo' (default). This feature will be removed in a future release. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.

PLAY ***************************************************************************

TASK [setup] *******************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "module_stderr": "sudo: a password is required\n", "module_stdout": "", "msg": "MODULE FAILURE", "parsed": false}

NO MORE HOSTS LEFT *************************************************************
	to retry, use: --limit @galaxy.retry

PLAY RECAP *********************************************************************
localhost                  : ok=0    changed=0    unreachable=0    failed=1   
```